lua/nvim-tree/legacy.lua

```diff
@@ -313,6 +321,9 @@ function M.migrate_legacy_options(opts)
   -- silently move
   refactored(opts)
 
+  -- silently clean unnecessary options that are now default behaviour
+  unnecessary(opts)
```

Comment:
Contemplating a mechanism of informing users, however users do get upset when messages are displayed on startup.

Perhaps some sort of one-shot mechanism or a way to disable messages.

Reply:
Perhaps something like `vim.notify_once()` could be used? I really dont mind those messages either, would rather know about changes I need to make than have broken setup.

Reply:
Today I Learned `notify_once`.

I was hopingi it persists across sessions, however sadly not.

#1667